src/cache_cf.cc

```diff
@@ -4428,13 +4414,12 @@ static void parse_sslproxy_cert_sign(sslproxy_cert_sign **cert_sign)
     while (*cert_sign)
         cert_sign = &(*cert_sign)->next;
 
-    *cert_sign = cs;
+    *cert_sign = cs.release();
 }
 
 static void dump_sslproxy_cert_sign(StoreEntry *entry, const char *name, sslproxy_cert_sign *cert_sign)
 {
-    sslproxy_cert_sign *cs;
-    for (cs = cert_sign; cs != NULL; cs = cs->next) {
+    for (auto *cs = cert_sign; cs; cs = cs->next) {
```

Comment:
If possible, just like in the other similar loop:

```suggestion
    for (const auto *cs = cert_sign; cs; cs = cs->next) {
```